server/base/src/main/java/org/apache/accumulo/server/util/ECAdmin.java

```diff
@@ -153,43 +165,115 @@ private void listCompactorsByQueue(ServerContext context) {
     }
   }
 
-  private void runningCompactions(ServerContext context, boolean details) {
+  private void runningCompactions(ServerContext context, boolean details, String format) {
     CompactionCoordinatorService.Client coordinatorClient = null;
-    TExternalCompactionList running;
+    Map<String,TExternalCompaction> runningCompactionsMap = new HashMap<>();
+
+    // Default to "plain" format if null or empty
+    if (format == null || format.trim().isEmpty()) {
+      format = "plain";
+    }
+
+    // Validate format
+    Set<String> validFormats = Set.of("plain", "csv", "json");
+    if (!validFormats.contains(format.toLowerCase())) {
+      throw new IllegalArgumentException(
+          "Invalid format: " + format + ". Expected: plain, csv, or json.");
+    }
```

Comment:
I don't think you need to re-validate the format if you are setting it to a known value.

```suggestion
    if (format == null || format.trim().isEmpty()) {
      format = "plain";
    } else {
    // Validate format
      Set<String> validFormats = Set.of("plain", "csv", "json");
      if (!validFormats.contains(format.toLowerCase())) {
        throw new IllegalArgumentException(
            "Invalid format: " + format + ". Expected: plain, csv, or json.");
      }
    }
```